src/bool/bvec2.rs

```diff
@@ -11,7 +11,12 @@ pub const fn bvec2(x: bool, y: bool) -> BVec2 {
 }
 
 /// A 2-dimensional `bool` vector mask.
-#[derive(Clone, Copy, PartialEq, Eq, Hash)]
+#[derive(Clone, Copy)]
```

Comment:
It would be preferable for the generated code to keep `#[derive(Clone, Copy, PartialEq, Eq, Hash)]` in a single derive, even if it means some duplication in the template. My focus is on making the generated code more readable, as that's what most people will see.